django/db/migrations/loader.py

```diff
@@ -122,6 +123,9 @@ def load_disk(self):
                         ) from e
                     else:
                         raise
+                # Force a reload if it's already loaded (tests need this)
+                if was_loaded:
+                    reload(migration_module)
```

Comment:
Similar block exists above.

Reply:
I thought better of this after some tests failed on postgis, where there are migrations that run queries for versions.